certbot/certbot/_internal/log.py

```diff
@@ -121,10 +124,13 @@ def post_arg_parse_setup(config):
         level = -config.verbose_count * 10
     stderr_handler.setLevel(level)
     logger.debug('Root logging level set at %d', level)
-    logger.info('Saving debug log to %s', file_path)
+
+    if not config.quiet:
```

Comment:
Just found a todo in the meeting notes saying that we should only print this when we encounter an error (so, at exit, probably?). Do we still think that's viable/correct?

Reply:
I tried that change, but ended up not keeping it.

My gut feel was that cutting this was going overboard on the minimization. The message is relatively useful and unobtrusive. I'm on the fence, since we also remind the user about it in the error postamble. 

I think if we remove this message, we should probably update the "See the logfiles in /var/log/letsencrypt" message  to point to the specific logfile, so the user doesn't have to go digging in the directory to find the most recent one.

Reply:
Ok, that sounds good to me, just wanted to make sure we weren't accidentally leaving it in.